templates/components/news-list/news-item-list.twig

```diff
@@ -1,97 +1,59 @@
 {#
 /**
  * @file
- * Card Component.
+ * News Banner.
  *
- * Basic building block component with the following variants:
- *   - Vertical (default): An image above text content
- *   - Horizontal: An image beside text content
- *   - Link: Whole card is a link with no button or call to action link
+ * Basic Page Banner for news.
  *
  * Available variables:
  * - attributes: For additional HTML attributes not already provided.
- * - modifier_class: Additional CSS classes to change look and behaviour.
- * - card_link: The href value for the card. A single card is only allowed to link to one URL.
- * - card_headline_link: Boolean. Set to true if headline acts as a link.
- * - card_image: A full HTML entity for an image.
- * - card_super_headline: Text super headline. Often used for metadata such as category.
- * - card_headline: Text heading
- * - card_body: An open variable for anything. Generally this would be for
- *     WYSIWYG editor contents.
- * - card_cta_attributes: Any html attributes on the link that need to be
- *     added. For example: _target. This acts as the attributes for the link button and headline link as well.
- * - card_cta_label: The clickable text for the CTA link
- * - card_button_label: The text label of the button component.
+ * - news_card_image: (needs description)
+ * - news_card_publishing_date: (needs description)
+ * - news_card_headline: (needs description)
+ * - news_card_dek: (needs description)
+ * - news_card_topics: (needs description)
+ * - news_source: (needs description)
  */
 #}
-
-{# For variant where whole card is a link, suppress all other links within the component #}
-{%- if "su-card--link" in modifier_class -%}
-  {%- set allow_links = false -%}
-{%- else -%}
-  {%- set allow_links = true -%}
-{%- endif -%}
-
-<ul class="su-news-list {{ modifier_class }}">
-  {# For variant where whole card is a link, add an <a> within the outer wrapper <div> #}
-  {%- if allow_links == false %}
-  <a href="{{ card_link }}" {{ card_cta_attributes }}>
-    {%- endif -%}
-
-    {# Card Image #}
-    {% if news_card_image is not empty -%}
+<ul {{ attributes }} class="su-news-list {{ modifier_class }}">
```

Comment:
@jenbreese @boznik The outer most wrapper must have the `{{ attributes }}` variable but when we add that in we open up the potential for duplicate `class` properties. This should be worked on to remove the `class` property from the tag and into `{{ attributes.addClass }}`

Reply:
@jenbreese assigned